### PR TITLE
Create reusable "include" for inserting data

### PR DESCRIPTION
**Pull request by umjennifer**
- create reusable html "include" for inserting data based on the column's mysql data type
- implement read for /election_years page (created to test the reusable html include)

### CODE REVIEW

File: what_the_fec/api/columns_information.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    return f"""
+        SELECT 
+            COLUMN_NAME, 
+            DATA_TYPE, 
+            CHARACTER_MAXIMUM_LENGTH, 
+            COLUMN_DEFAULT, 
+            IS_NULLABLE 
+            FROM INFORMATION_SCHEMA.COLUMNS
+            WHERE table_name = '{table_name}' AND COLUMN_NAME != "id"
+        """
```
Comment (nolandseigler) [inline what_the_fec/api/columns_information.py:12]: we should do our best not to f string table_name in and parametrize the query later after putting the placeholder for table_name. I know we have some original ones that are like this that we need to cleanup I just hope that number does not grow.

File: what_the_fec/templates/includes/insert_fieldset.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<div id="insert" style="display: block">
+    <form id="insert_{{table_name}}" action="/{{table_name}}" method="post">
+            <fieldset>
+                <legend><strong>Add item to {{table_name}}</strong></legend>
+
+                {% for col_name in items[0].keys() %}
```
Comment (nolandseigler) [inline what_the_fec/templates/includes/insert_fieldset.html:6]: this whole file is really awesome. great work.

File: what_the_fec/api/columns_information.py (hunk 1 above)
Comment (nolandseigler) [reply what_the_fec/api/columns_information.py:12]: also pulling this sql into its own functions is a great call it is in my plan to pull some of this down into the storage layer if time permits. worst case I will likely pull it into functions in the api(soon to be renamed to routes) package.

File: what_the_fec/templates/includes/insert_fieldset.html (hunk 2 above)
Comment (umjennifer) [reply what_the_fec/templates/includes/insert_fieldset.html:6]: Woot, thanks!

File: what_the_fec/api/columns_information.py (hunk 1 above)
Comment (umjennifer) [reply what_the_fec/api/columns_information.py:12]: Per our chat, let's put this in a TODO for the future. I'll add a TODO note.

**Review by nolandseigler — APPROVED** [review]

**Pull request merged**